lib/msf/core/handler/reverse_http.rb

```diff
@@ -99,7 +99,7 @@ def payload_uri(req)
     "#{scheme}://#{callback_host}/"
   end
 
-  # Use the {#refname} to determine whether this handler uses SSL or not
+  # Use the refname to determine whether this handler uses SSL or not
```

Comment:
This was right before.  `{#refname}` makes a link to the `refname` instance method.